features/account/sign_in.feature

```diff
@@ -24,3 +24,13 @@ Feature: Sign in to the store
         And I log in
         Then I should be notified about bad credentials
         And I should not be logged in
+
+    @todo
+    Scenario: A customer cannot log in after deleting their account
```

Comment:
Second one :+1:. The first one may suggest that it was deleted by the customer himself. :)